courses/B.yaml

```diff
@@ -26,7 +26,11 @@
   desc: |
     LaTeX is the standard method for making publications in the
     computational and physical sciences.
-
+  reading: |
+    Short LaTeX `tutorial <http://andre-geldenhuis.github.io/Latex-Short/>`_ based on Software Carpentry 
+  video: |
+    `LaTeX tutorial series <https://www.youtube.com/watch?v=FXujG7c9p8g>`_ covering LaTeX formatting and constructing a report
```

Comment:
```suggestion
    `LaTeX tutorial series <https://www.youtube.com/playlist?list=PLNnwglGGYoTtW7o4PHFOSWGevcdFa3v3D>`_ covering LaTeX formatting and constructing a report
```
Linked to full playlist (took some searching...).  The video series seems really nice!  Really good.